src/Compilers/Core/Portable/Compilation/NullableContext.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents the state of the nullable analysis at a specific point in a file.
+    /// </summary>
+    [Flags]
+    public enum NullableContext
+    {
+        /// <summary>
+        /// Nullable warnings and annotations are off at this location.
```

Comment:
Not sure if this needs some name to clarify it's also not being inherited or something? It otherwise seems like somebody might naviely write nullableContext == NullableContext.Disabled and that's *wrong*.

Reply:
So that's a good point, but we also want to keep the names consistent with the `#nullable` syntax, which uses the word `disable`. Perhaps we should just update the documentation to call that out explicitly?